Keypad/Keypad.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable react/jsx-no-bind */
+
+/*
+ * A keypad used to display a sequence of numbers and buttons, like a keyboard.
+ */
```

Comment:
Please use `/** */` format if this comment is used for the JS Doc.

In addition, please follow up the comment rule like the `Item`.
```
/**
 * Provides Agate-themed item components and behaviors. Useful for content in lists.
 *
 * @example
 * <Item>Hello Enact!</Item>
 *
 * @module agate/Item
 * @exports Item
 * @exports ItemBase
 * @exports ItemDecorator
 */
```